src/app/zones/zone/zone-detail.component.html

```diff
@@ -0,0 +1,156 @@
+<div *ngIf="zone" class="surface-card p-4 shadow-2 border-round mb-4">
+  <div class="flex flex-row justify-content-between">
+    <div class="text-3xl font-medium text-900 mb-3" data-cy="zone-name">
+      <a
+        *ngIf="!isSingle"
```

Comment:
I don't really like the name `isSingle` because it doesn't say what is being changed. If the component is `Single`, then it has no detail link, do I see that right? If yes, then the name should be something in this direction.

Reply:
good point, I've refactored that